### PR TITLE
feat(components): add Panel and Box

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/SURF-746

* Added Panel component
* Added Box component

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### Box Component
box-demo

box-after

### Panel Component
panel-demo
panel-after

### CODE REVIEW

File: source/components/panel/index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              </label>
+            </p>
+          </hx-panelbody>
+        </hx-panel>
+      </div>
+    </div>
+    <br />
+    {% code 'html' %}
+      <hx-panel>
+        [<hx-panelhead></hx-panelhead>]
```
Comment (cathysiller) [inline source/components/panel/index.html]: are the `[]` around the head and footer for optional? will users naturally know that?

File: source/components/panel/panel-docs.less [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  min-height: 200px;
+  overflow: hidden;
+  resize: vertical;
+
+  hx-panel {
+    border: 1px dotted black;
+  }
+
+  hx-panelhead {
+    background-color: @purple-900;
```
Comment (cathysiller) [inline source/components/panel/panel-docs.less]: are the background colors approved by design? are these just used for examples and user can change?

Comment (CITguy) [reply source/components/panel/panel-docs.less]: Background colors are purely for documentation. No need for design sign-off.

File: source/components/panel/index.html (hunk 1 above)
Comment (CITguy) [reply source/components/panel/index.html]: This is common syntax that many developers have likely seen. It's used in man pages for CLI programs and it's also used in the [EncoreUI ngdocs](http://rackerlabs.github.io/encore-ui/5.x/ngdocs/index.html#/api/elements.directive:rxBulkSelect).

Comment (CITguy) [reply source/components/panel/index.html]: If there is confusion, we can address it as it is brought to light.

**Comment by Droogans** [comment]
Oh boy this is going to speed up the need for us to get to https://github.com/rackerlabs/snappit-visual-regression/issues/27

File: source/components/box/box-demo.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+if (document.getElementById('vue-boxDemo')) {
+    new Vue({
+        el: '#vue-boxDemo',
+        data: {
+            hello: 'Hello from VueJS!',
```
Comment (technabors) [inline source/components/box/box-demo.js]: Remove test.

File: source/components/panel/index.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                application like the VOC branching. The best way to document acceptance
+                impediment is to periodically build their meeting. It was discovered
+                that by efficiently releasing the MVP products, we can practice the
+                lean XP development team in the minor board.
+              </p>
+            </hx-panelbody>
+            <hx-panelfoot v-if="hasFoot">Panel Foot</hx-panelfoot>
+          </hx-panel>
+        </div>
+        <small class="hxItalic">
```
Comment (technabors) [inline source/components/panel/index.html]: Double-check within the typography file to see if caption styling can be used instead.

**Comment by technabors** [comment]
zoom LGTM

**Pull request merged**